source/guides/stacks/dlrs.rst

```diff
@@ -998,7 +909,7 @@ Related topics
 
 .. _DLRS Terms of Use: https://clearlinux.org/stacks/deep-learning/terms-of-use
 
-.. _DLRS Release notes: https://github.com/clearlinux/dockerfiles/blob/master/stacks/dlrs/releasenote.md
+.. _DLRS Release notes: https://github.com/intel/stacks/tree/master/dlrs
```

Comment:
Hey @bd-dean. Just checking. Do I understand correctly that the purpose is to link to release notes only on GitHub? 
I believe this is so. If that's so, then do you want to link directly to the release notes? Like so: 
https://github.com/intel/stacks/blob/master/dlrs/clearlinux/releasenote.md